components/arm/client.go

```diff
@@ -67,6 +67,9 @@ func (c *client) EndPosition(ctx context.Context, extra map[string]interface{})
 	if err != nil {
 		return nil, err
 	}
+	if resp.Pose == nil {
```

Comment:
hmmm, I tend to like only passing the most basic logic in the client code, and tend to remove things that enforce any checks or logic here, I think an appropriate place for these warnings is in `arm.go/CreateStatus` to check if the jointpositions and modelframe being passed to ComputeOOBPosition are not nil, but maybe I am missing something here?

Reply:
I tend to agree with Rand here.  I'm also not sure if warnings are the thing to do here in general.  If its problematic to the point of causing actual problems we should be erroring and if it's not a problem theres no reason to be noisier than we need to

Reply:
Makes sense - I was on the fence about putting it here anyways and am happy to move it to `arm.go/CreateStatus`

Reply:
On second thought, I think it might be best to put the checks into `ComputeOOBPosition` within `motionplan/kinematic.go` since that function is called in many places. Does that sound good?

Reply:
fine with me, just give the error messages a lot of clarity for a novice to understand.

Reply:
There seems to be disagreement about whether or not we should throw a warning additionally to returning an error. For now, I made the requested changes but kept the warnings. You can see the logging results in the description on top of this PR.

I vote for adding the warnings since they're easier to read and provide the location of where the error is thrown, which makes debugging easier. On the other hand, I am not sure if the changes I made to achieve that follow a good pattern - if not, then I'm happy to remove the warnings and keep the error messages only.